code/app/src/main/java/com/example/the_tarlords/MainActivity.java

```diff
@@ -119,7 +119,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
                             //creates 'user' object from firestore data, now you can use 'user' object
                             user = documentSnapshot.toObject(User.class);
-
+                            // needs to be above the setBinding()
+                            isAdmin = user.getIsAdmin();
```

Comment:
will this be a problem if it returns null? (ie the firestore doc doesn't have a getIsAdmin field)